### DISCUSSION TITLE
Mirth Connect Administrator install on Ubuntu 20.04 LTS?

### DISCUSSION THREAD

**Discussion by matthornish** [nextgenhealthcare/connect#5069] [Q&A] [answered]
Hello, everyone,

I have a Mirth instance (version 3.8.1) running on a Windows server.  I'm looking to install the Mirth Connect Administrator on my Ubuntu 20.04 LTS client PC.  However, when I go to the URL of the server for downloading the Administrator Launcher app, clicking the download button just redirects me to a page with what looks like a BASH script followed by a bunch of gibberish.

I like to avoid logging directly into servers whenever possible, so I am hoping someone out there has some wisdom to share with me regards how to get this working.

Thanks in advance,

Matt

**Comment by jonbartels** [comment, marked as answer]
You actually probably want to install the MC Admin Launcher. The Launcher deals with caching, code signature verification, etc. and it provides a handy server listing for MC instances too. You can download it directly from - https://www.nextgen.com/products-and-services/mirth-connect-downloads 

If you don't want to use the Launcher, you can launch MC administrator with `javaws http://yourserver:8080/webstart.jnlp` .
